app/models/measurement.rb

```diff
@@ -10,11 +9,8 @@ class Measurement < ActiveRecord::Base
 
   #input: measurement
   def bmi_calc
```

Comment:
This was an assignment of one of the other coaches. Initially, I had a bmi column in my database.  This is an exercise to differentiate between persistent and process data, where the process data may not be stored in the database itself - at least as long as I am not working on nu.nl or something. Also, it was an exercise in the Rails way of where to put things, with the fat models and the skinny controllers.